roles/matrix-base/templates/static-files/well-known/matrix-client.j2

```diff
@@ -26,12 +26,27 @@
 		"preferredDomain": {{ matrix_client_element_jitsi_preferredDomain|to_json }}
 	}
 	{% endif %}
-	{% if not matrix_client_element_e2ee_default %},
-	"io.element.e2ee": {
-		"default": false
+	,
+        "io.element.e2ee": {
+        	{% if not matrix_client_element_e2ee_default %}
+		   "default": false
+		{% else %}
+		   "default": true
+                {% endif %}
+		{% if matrix_client_element_e2ee_backup_required %},
+	           "secure_backup_required": true
+		{% else %}
+	           "secure_backup_required": false
+         	{% endif %}
+		{% if matrix_client_element_e2ee_backup_methods %},
```

Comment:
Looking at the [E2EE docs for Element](https://github.com/vector-im/element-web/blob/develop/docs/e2ee.md):

> If the secure_backup_setup_methods field is not present or exists but does not contain any supported methods, Element will fallback to the default value of: ["key", "passphrase"].

So, it looks like always defining this key may be simpler than conditionally defining it. We can probably remove this `if` statement.

By the way, indentation seems to be off somehow.

Reply:
same here, but I also updated the defaults/main.yml to contain `[ "key", "passphrase" ]`, I'll push this at least.

Reply:
Leaving it empty is probably better, because (as per the above message) empty or missing means: use "whatever its default is" (which is currently `["key", "passphrase"]` but may change later).

Reply:
ok, will change that back.